src/axom/spin/tests/CMakeLists.txt

```diff
@@ -43,15 +43,16 @@ foreach ( test ${spin_tests} )
 
 endforeach()
 
-if ( RAJA_FOUND AND UMPIRE_FOUND )
 
-    blt_add_executable(
-      NAME       spin_bvh_test
-      SOURCES    spin_bvh.cpp
-      OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
-      DEPENDS_ON ${spin_depends}
-      FOLDER     axom/spin/tests
-      )
+blt_add_executable(
+  NAME       spin_bvh_test
+  SOURCES    spin_bvh.cpp
+  OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+  DEPENDS_ON ${spin_depends}
+  FOLDER     axom/spin/tests
+  )
+
+if ( RAJA_FOUND AND UMPIRE_FOUND )
 
     axom_add_test(
```

Comment:
Consider calling `axom_add_test` outside the test `if ( RAJA_FOUND AND UMPIRE_FOUND )`, so that the only thing inside the test is the call to `set_property(... "CUDA_VISIBLE_DEVICES=0")`

Reply:
Done 👍